### ISSUE TITLE
implementation details for ACDC

### ISSUE THREAD

**Issue by kathyliu579** [Beckschen/TransUNet#91]
i use the same paramater like SGD, lr  for ACDC. but the loss can not be reduced when loss = 0.2
Please tell me what are your training parameters?

**Comment by zhouyi-git** [comment]
Hello, can you tell me how to configure ACDC dataset? I want to reproduce the code, my email 1092622009@qq /Thank you for any help.

**Comment by guangguangLi** [comment]
> i use the same paramater like SGD, lr for ACDC. but the loss can not be reduced when loss = 0.2 Please tell me what are your training parameters?

Hello, can you share the preprocessed ACDC dataset? I want to reproduce the code, my email is 1120639437@qq.com. Thanks for your help.

**Comment by guangguangLi** [comment]
> Hello, can you tell me how to configure ACDC dataset? I want to reproduce the code, my email 1092622009@qq /Thank you for any help.

Hello, can you share the preprocessed ACDC dataset? I want to reproduce the code, my email is 1120639437@qq.com. Thanks for your help.

**Comment by Gi-gigi** [comment]
> > Hello, can you tell me how to configure ACDC dataset? I want to reproduce the code, my email 1092622009@qq /Thank you for any help.
> 
> Hello, can you share the preprocessed ACDC dataset? I want to reproduce the code, my email is [1120639437@qq.com](mailto:1120639437@qq.com). Thanks for your help.

Hello, can you share the preprocessed ACDC dataset? I want to reproduce the code, my email is [565290560@qq.com](mailto:1120639437@qq.com). Thanks for your help.

**Comment by laodetangkoushui** [comment]
> Hello, can you share the preprocessed ACDC dataset? I want to reproduce the code, my email is [565290560@qq.com](mailto:1120639437@qq.com). Thanks for your help

Hello, can you share the preprocessed ACDC dataset? I want to reproduce the code, my email is 1786225568@qq.com. Thanks for your help

**Comment by kathyliu579** [comment]
Hi both i share the code in my paper and github you can cite it and get it from

https://github.com/kathyliu579/CS-Unet/tree/main

**Comment by laodetangkoushui** [comment]
Thank you very much for providing this link. I will quote your article when my paper is finished




郑昀晖
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Beckschen/TransUNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月25日(星期天) 晚上11:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Beckschen/TransUNet] implementation details for ACDC (Issue #91)





 
Hi both i share the code in my paper and github you can cite it and get it from
 
https://github.com/kathyliu579/CS-Unet/tree/main

**Comment by laodetangkoushui** [comment]
I found that the source code provided on github does not have a data set, can you provide a data set for me


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Beckschen/TransUNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月25日(星期天) 晚上11:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Beckschen/TransUNet] implementation details for ACDC (Issue #91)





 
Hi both i share the code in my paper and github you can cite it and get it from
 
https://github.com/kathyliu579/CS-Unet/tree/main
